go/vt/vtgate/evalengine/internal/json/helpers.go

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
do we really want to add these license headers to all files? feels weird, but I have no idea what is correct

Reply:
I think so, since we've (significantly) modified these files. The idea is we add (C) for the original author, then the Vitess contributors (C).